javascript/portfolio.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 var portObjectArray = [];
```

Comment:
I would suggest refactoring this so it is part of the object. You will then need to refactor code wherever you use this.